dandiapi/analytics/migrations/0002_remove_processeds3log_analytics_processeds3log_unique_name_embargoed_and_more.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.1.13 on 2024-02-23 00:16
+from __future__ import annotations
+
+from django.db import migrations, models
+
+
+def remove_embargoed_objects(apps, _):
+    ProcessedS3Log = apps.get_model('analytics.ProcessedS3Log')
+    ProcessedS3Log.objects.filter(embargoed=True).delete()
```

Comment:
I guess I don't see the point, since we'd need to create a new model relation to achieve that, and afaik `ProcessedS3Log` is only used to track download counts. There's a chance that as a part of the migration, we'll miss a few logs, but imo I don't think that's a big deal. It would just result in the download counts for certain embargoed blobs being slightly off.

Your comment made me wonder about if anything will go awry by deleting these embargoed `ProcessedS3Log` objects, since as a part of ingressing new records, we look at the latest entries by name. However, I don't think that'll be an issue, since prior to this PR, those tasks were run separately for embargoed and non-embargoed records. So it seems that the main adverse effect would be that if we ever needed to fully recalculate download counts based on s3 logs, we wouldn't have those records. They would still exist in the old embargoed log bucket, but I'm honestly not sure what we would do to inspect them if that came up.

Another option is to simply migrate those logs from the embargoed log bucket to the main log bucket. Assuming there's no conflicts with object names, that would be a fairly trivial task, albeit time consuming with ~350,000 objects in the embargoed log bucket.

Maybe @danlamanna has some thoughts on this.

Reply:
After further offline discussion we've decided to keep these embargoed records around for the time being. I'll work on updating the implementation to allow for this.